src/equations/compressible_euler_2d.jl

```diff
@@ -970,6 +970,46 @@ end
   return SVector(f1m, f2m, f3m, f4m)
 end
 
+@inline function splitting_lax_friedrichs(u, ::Val{:plus}, normal_direction::AbstractVector,
+                                          equations::CompressibleEulerEquations2D)
+  rho, rho_v1, rho_v2, rho_e = u
+  v1 = rho_v1 / rho
+  v2 = rho_v2 / rho
+  v_dot_n = normal_direction[1] * v1 + normal_direction[2] * v2
+  p = (equations.gamma - 1) * (rho_e - 0.5 * (rho_v1 * v1 + rho_v2 * v2))
+
+  a = sqrt(equations.gamma * p / rho)
+  H = (rho_e + p) / rho
+  lambda = 0.5 * (v_dot_n + a * norm(normal_direction))
+
+  f1p = 0.5 * rho * v_dot_n + lambda * u[1]
+  f2p = 0.5 * rho * v_dot_n * v1 + 0.5 * p * normal_direction[1] + lambda * u[2]
+  f3p = 0.5 * rho * v_dot_n * v2 + 0.5 * p * normal_direction[2] + lambda * u[3]
+  f4p = 0.5 * rho * v_dot_n * H + lambda * u[4]
+
+  return SVector(f1p, f2p, f3p, f4p)
+end
```

Comment:
This is not the same as rotating to x, applying the Cartesian splitting, and rotating back:
```julia
julia> using Revise; using Trixi

julia> equations = CompressibleEulerEquations2D(1.4)
┌──────────────────────────────────────────────────────────────────────────────────────────────────┐
│ CompressibleEulerEquations2D                                                                     │
│ ════════════════════════════                                                                     │
│ #variables: ………………………………………………… 4                                                                │
│ │ variable 1: …………………………………………… rho                                                              │
│ │ variable 2: …………………………………………… rho_v1                                                           │
│ │ variable 3: …………………………………………… rho_v2                                                           │
│ │ variable 4: …………………………………………… rho_e                                                            │
└──────────────────────────────────────────────────────────────────────────────────────────────────┘

julia> u = SVector(1.0, 0.2, -0.3, 2.0)
4-element SVector{4, Float64} with indices SOneTo(4):
  1.0
  0.2
 -0.3
  2.0

julia> normal_direction = SVector(0.3, -0.5)
2-element SVector{2, Float64} with indices SOneTo(2):
  0.3
 -0.5

julia> let
           norm_ = Trixi.norm(normal_direction)
           normal_vector = normal_direction / norm_
           
           u_rotated = Trixi.rotate_to_x(u, normal_vector, equations)
           f = splitting_lax_friedrichs(u_rotated, Val{:plus}(), 1, equations)
           f_rotated = Trixi.rotate_from_x(f, normal_vector, equations) * norm_

           f_rotated - splitting_lax_friedrichs(u, Val{:plus}(), normal_direction, equations)
       end
4-element SVector{4, Float64} with indices SOneTo(4):
  0.00011898020814316013
  2.3796041628665332e-5
 -3.5694062443025754e-5
  0.0002379604162865423
```
You mentioned some unusual behavior required for the splittings in general. Is this behavior what we want/need?